agent/docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
   # Otherwise, MySQL in Docker for Mac completely locks during/after slowlog rotation tests.
   mysql:
     image: ${MYSQL_IMAGE:-percona:5.7}
+    platform: linux/amd64
```

Comment:
can you make it configurable, so that when MYSQL will support arm64 it can be overriden

Reply:
Once MySQL will support arm64 what is the reason to have this field at all? I guess we will just drop it, no?

Reply:
What I am trying to say, it is always better to avoid hard codding.

Reply:
And the title of the PR misleading, because it is not a fix but rather a workaround.
It can extra details/motivation, ie "On arm64 mac machines, running mysql fails with `no matching manifest for linux/arm64/v8 in the manifest list entries`"

I just checked upstream, and I see that arm64 is supported (ie in mysql:8.0.29-oracle)
![image](https://user-images.githubusercontent.com/91831719/177049978-f17131ac-129e-4fbf-909b-c8a8b8c1a8c2.png)